clang/test/CodeGen/function_overload_crash.c

```diff
@@ -0,0 +1,7 @@
+// RUN: %clang_cc1 -emit-llvm < %s
```

Comment:
Can you integrate this test into some existing file that's testing something similar?

Please leave the bug number in a comment: something like `//GH88917`.

Please use FileCheck CHECK lines to verify the correct call instruction is emitted.

Reply:
Hello, I apologize, but could I ask some pointers to some docs for creating tests for llvm? I didn't find any very good resource about it :)

Reply:
Have you seen https://llvm.org/docs/TestingGuide.html#regression-test-structure ?

If you have any ideas for improving the documentation, please let me know (or better, propose a patch; it's generated from llvm/docs/TestingGuide.rst).

Reply:
Yes. I gave another read now and found out the FileCheck guide to solve some of my doubts. 
I will need to double check also the llvm IR generation, if everything is ok, I should be able to modify the test quickly.
As of now I am for a bit more than a week away from pc, but as soon as I am back I will finish this pr.

thank you for your help, I'll be posting soon :)